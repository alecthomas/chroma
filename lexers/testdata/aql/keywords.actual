LET
AT LEAST
NULL
nULL
true
false
TRUE
FALSE
True
fAlSe
WITH COUNT INTO
KEEP
prune
SEARCH
to
OPTIONS {}
options{}
CURRENT
NEW
OLD

LEAST
AT
CAT LEAST
KNULL
falsey
COUNT
OPTIONS
current
New
old