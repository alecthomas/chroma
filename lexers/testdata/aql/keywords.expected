[
  {"type":"KeywordDeclaration","value":"LET"},
  {"type":"Text","value":"\n"},
  {"type":"KeywordReserved","value":"AT LEAST"},
  {"type":"Text","value":"\n"},
  {"type":"KeywordConstant","value":"NULL"},
  {"type":"Text","value":"\n"},
  {"type":"KeywordConstant","value":"nULL"},
  {"type":"Text","value":"\n"},
  {"type":"KeywordConstant","value":"true"},
  {"type":"Text","value":"\n"},
  {"type":"KeywordConstant","value":"false"},
  {"type":"Text","value":"\n"},
  {"type":"KeywordConstant","value":"TRUE"},
  {"type":"Text","value":"\n"},
  {"type":"KeywordConstant","value":"FALSE"},
  {"type":"Text","value":"\n"},
  {"type":"KeywordConstant","value":"True"},
  {"type":"Text","value":"\n"},
  {"type":"KeywordConstant","value":"fAlSe"},
  {"type":"Text","value":"\n"},
  {"type":"KeywordReserved","value":"WITH"},
  {"type":"Text","value":" "},
  {"type":"KeywordPseudo","value":"COUNT"},
  {"type":"Text","value":" "},
  {"type":"KeywordReserved","value":"INTO"},
  {"type":"Text","value":"\n"},
  {"type":"KeywordPseudo","value":"KEEP"},
  {"type":"Text","value":"\n"},
  {"type":"KeywordPseudo","value":"prune"},
  {"type":"Text","value":"\n"},
  {"type":"KeywordPseudo","value":"SEARCH"},
  {"type":"Text","value":"\n"},
  {"type":"KeywordPseudo","value":"to"},
  {"type":"Text","value":"\n"},
  {"type":"KeywordPseudo","value":"OPTIONS"},
  {"type":"Text","value":" "},
  {"type":"Punctuation","value":"{}"},
  {"type":"Text","value":"\n"},
  {"type":"KeywordPseudo","value":"options"},
  {"type":"Punctuation","value":"{}"},
  {"type":"Text","value":"\n"},
  {"type":"NameBuiltinPseudo","value":"CURRENT"},
  {"type":"Text","value":"\n"},
  {"type":"NameBuiltinPseudo","value":"NEW"},
  {"type":"Text","value":"\n"},
  {"type":"NameBuiltinPseudo","value":"OLD"},
  {"type":"Text","value":"\n\n"},
  {"type":"Name","value":"LEAST"},
  {"type":"Text","value":"\n"},
  {"type":"Name","value":"AT"},
  {"type":"Text","value":"\n"},
  {"type":"Name","value":"CAT"},
  {"type":"Text","value":" "},
  {"type":"Name","value":"LEAST"},
  {"type":"Text","value":"\n"},
  {"type":"Name","value":"KNULL"},
  {"type":"Text","value":"\n"},
  {"type":"Name","value":"falsey"},
  {"type":"Text","value":"\n"},
  {"type":"Name","value":"COUNT"},
  {"type":"Text","value":"\n"},
  {"type":"Name","value":"OPTIONS"},
  {"type":"Text","value":"\n"},
  {"type":"Name","value":"current"},
  {"type":"Text","value":"\n"},
  {"type":"Name","value":"New"},
  {"type":"Text","value":"\n"},
  {"type":"Name","value":"old"}
]
